apps/desks.py

```diff
@@ -537,6 +540,24 @@ def _find_other_sluglines(self, family_id, slugline, versioncreated, desk_id):
         return (False, older_sluglines)
 
 
+class OverviewDesksResource(Resource):
+    url = r'desks/<regex("([a-f0-9]{24})|all"):desk_id>/overview'
+    resource_title = "desk_overview"
+    resource_methods = ['GET']
+
+
+class OverviewDesksService(BaseService):
+
+    def on_fetched(self, doc):
+        desk_id = request.view_args['desk_id']
+        desk_match = {"$ne": None} if desk_id == "all" else ObjectId(desk_id)
+        aggregation = app.data.pymongo().db.archive.aggregate([
+            {"$match": {"task.desk": desk_match}},
```

Comment:
it might need some state filter here, otherwise it also counts published items (which might be fine, just not sure)

Reply:
I was counting published filter on purpose, but you're right that I don't if it's need. @pavlovicnemanja should we count them in the summary?

Reply:
![image](https://user-images.githubusercontent.com/6507688/85305952-1520c600-b4ae-11ea-8492-542f7af14e91.png)

We need total items per stage. Grand Total of all stages in desk can be done on frontend part.

Reply:
so to be clear, it's including published items right? Nothing to change then.